src/interfaces/import-account-type.ts

```diff
@@ -0,0 +1,24 @@
+export enum ImportAccountTypeEnum {
+  PRIVATE_KEY = 'PRIVATE_KEY',
+  SEED_PHRASE = 'SEED_PHRASE'
+}
+
+export enum ImportAccountDerivationEnum {
+  DEFAULT = 'DEFAULT',
+  CUSTOM_PATH = 'CUSTOM_PATH'
+}
+
+export type ImportAccountTypeValues = {
+  type: ImportAccountTypeEnum;
+};
+
+export type ImportAccountSeedValues = {
+  seedPhrase: string;
+  password?: string;
+  derivation?: string;
```

Comment:
why `derivation` is optional and why not `ImportAccountDerivationEnum` type?
also maybe rename to `derivationType`?